builder/dockerignore/dockerignore.go

```diff
@@ -43,7 +43,7 @@ func ReadAll(reader io.Reader) ([]string, error) {
 		excludes = append(excludes, pattern)
 	}
 	if err := scanner.Err(); err != nil {
-		return nil, fmt.Errorf("Error reading .dockerignore: %v", err)
+		return nil, fmt.Errorf("Error reading dockerignore file: %v", err)
```

Comment:
Can we add the name of the dockerignore file to this error message to help to user when they "fat finger" it?